R/readWriteMgfData.R

```diff
@@ -83,6 +102,11 @@ writeMgfContent <- function(sp, TITLE = NULL, con) {
         }
     } else .cat("\nRTINSECONDS=", rtime(sp))
 
+    if (length(addFields)) {
```

Comment:
Instead of the nested `if` statement: `length(addFields) && !is.null(names(addFields))` or `is.vector(addFields) && !is.null(names(addFields))`?

Reply:
good point. Will change.